internals/cli/cli.go

```diff
@@ -242,11 +237,6 @@ func Parser(cli *client.Client) *flags.Parser {
 	return parser
 }
 
-func applyPersonality(s string) string {
-	r := strings.NewReplacer("{{.ProgramName}}", cmd.ProgramName, "{{.DisplayName}}", cmd.DisplayName)
-	return r.Replace(s)
-}
```

Comment:
Decided to move this to the `cmd` package (so it sits next to the variables it reads) and make it public. `cmd.ProgramName` and `cmd.DisplayName` were already public, so making it possible to use `ApplyPersonality()` from public code shouldn't hurt.